### PR TITLE
Add multi-cloud user guide

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind documentation

**What this PR does / why we need it**:
The PR adds multi-cloud documentation in the User Guides Section

**Which issue(s) this PR fixes**:

Fixes #1072 multi-cloud issue

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

**Test Report**:

image

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@kumarashit  @anvithks  Please review and merge

**Review by anvithks — COMMENTED** [review]
I think this is more of a developer guide a we are providing instructions to install from source and bring up Multi-cloud alone to setup an environment for development. 
@kumarashit @himanshuvar Please take a look.

**Comment by rajat-soda** [comment]
> I think this is more of a developer guide a we are providing instructions to install from source and bring up Multi-cloud alone to setup an environment for development.
> @kumarashit @himanshuvar Please take a look.

moved multi-cloud dev guide to Developer Guide Section

File: content/guides/developer-guides/object-lifecycle-management/_index.md [hunk 1]
```diff
@@ -7,26 +7,26 @@ tags: ["developer guide", "object lifecycle"]
 ---
 ## Overview
 
-This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using OpenSDS. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management [user guide](/guides/user-guides/object-lifecycle-management/) and Object Lifecycle [design spec](https://github.com/opensds/design-specs/blob/master/specs/capri/object_lifecycle_management.md). 
+This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using SODA. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management [user guide](/guides/user-guides/object-lifecycle-management/) and Object Lifecycle [design spec](https://github.com/opensds/design-specs/blob/master/specs/capri/object_lifecycle_management.md).
```
Comment (kumarashit) [inline content/guides/developer-guides/object-lifecycle-management/_index.md:10]:
```suggestion
This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using SODA. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management [user guide](/guides/user-guides/object-lifecycle-management/) and Object Lifecycle [design spec](https://github.com/sodafoundation/design-specs/blob/master/specs/capri/object_lifecycle_management.md). 
```

File: content/guides/developer-guides/object-lifecycle-management/_index.md [hunk 2]
```diff
@@ -7,26 +7,26 @@ tags: ["developer guide", "object lifecycle"]
...
-APIs are a set of functions and procedures that allow for the creation of applications that access data and features of other applications, services or operating system. In OpenSDS lifecycle management feature we are using APIs to communicate with different storage backend.
+APIs are a set of functions and procedures that allow for the creation of applications that access data and features of other applications, services or operating system. In SODA lifecycle management feature we are using APIs to communicate with different storage backend.
 
 
 ### Write new API
 Before starting to write the APIs, it is required to have basic understanding of XML and JSON syntax.
 
 #### There are three major steps involved:
-    a) All APIs go into the 's3' module of the OpenSDS multi-cloud package, i,e. opensds/multi-cloud/api/pkg/s3 directory
+    a) All APIs go into the 's3' module of the SODA multi-cloud package, i,e. opensds/multi-cloud/api/pkg/s3 directory
```
Comment (kumarashit) [inline content/guides/developer-guides/object-lifecycle-management/_index.md:21]:
```suggestion
    a) All APIs go into the 's3' module of the SODA multi-cloud package, i,e. sodafoundation/multi-cloud/api/pkg/s3 directory
```

File: content/guides/developer-guides/object-lifecycle-management/_index.md [hunk 3]
```diff
@@ -7,26 +7,26 @@ tags: ["developer guide", "object lifecycle"]
...
 =======
 ## Write new API
 Before starting to write the APIs, it is required to have basic understanding of XML and JSON syntax.
 
 ## How to write the APIs
 
 ### There are three major steps involved:
-    a) All APIs goes into the 's3' module of the OpenSDS multi-cloud package, i,e. opensds/multi-cloud/api/pkg/s3 directory
+    a) All APIs goes into the 's3' module of the SODA multi-cloud package, i,e. opensds/multi-cloud/api/pkg/s3 directory
```
Comment (kumarashit) [inline content/guides/developer-guides/object-lifecycle-management/_index.md:29]:
```suggestion
    a) All APIs goes into the 's3' module of the SODA multi-cloud package, i,e. sodafoundation/multi-cloud/api/pkg/s3 directory
```

File: content/guides/user-guides/object-lifecycle-management/_index.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+By following this guide you will be able to install SODA multi-cloud with creating backends and buckets, uploading object and setting up the lifecycle for those objects.
 
-## Installing Opensds multi-cloud
+## Installing SODA multi-cloud
 
-### Download opensds-installer code
+### Download SODA-installer code
 
 ```cassandraql
 git clone https://github.com/opensds/opensds-installer.git
```
Comment (kumarashit) [inline content/guides/user-guides/object-lifecycle-management/_index.md:21]:
```suggestion
git clone https://github.com/sodafoundation/installer.git
```

File: content/guides/user-guides/orchestration/_index.md [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
+SODA is aimed at addressing the storage integration challenges of both the Cloud Native environmnet and traditional IT environment.In a distributed environment, the business processes consist of several interconnected steps which may require streamline the process. SODA provides orchestrating these process to ensure accurate and faster execution with minimal manual intervention. The Orchestration framework provides flexibility to use existing workflows or define customized workflows to get the simplied execution of tasks. 
 
 ## Intended Readers
-OpenSDS users who plan to orchestrate complex tasks in a distributed environment
+SODA users who plan to orchestrate complex tasks in a distributed environment
 
 ## Purpose
 
 ## Problem reporting instructions
 Please refer to this guide or release notes first for any known issues or workarounds available. Issues related to Orchestration can be raised at https://github.com/opensds/orchestration/issues
```
Comment (kumarashit) [inline content/guides/user-guides/orchestration/_index.md:18]:
```suggestion
Please refer to this guide or release notes first for any known issues or workarounds available. Issues related to Orchestration can be raised at https://github.com/sodafoundation/orchestration/issues
```

File: content/guides/user-guides/orchestration/_index.md [hunk 6]
```diff
@@ -1,44 +1,44 @@
...
+SODA Capri release provides pre-defined service catalogs. Service catalogs serves specific purpose and abstracts the underlying actions from end-user. End user can create instances of these services and check the status. User can see the entities through SODA dashboard or use SODA CLIs to check it from console.
+SODA Orchestration service acts as the facilitator to orchestrate actions between SODA dashboard, Orchestration Manager and the SODA backend. Currently SODA supports StackStorm (https://docs.stackstorm.com/overview.html) as the Orchestration Manager. It uses Mistral workflows in StackStorm.
 
 ## Installation
-Please refer https://github.com/opensds/orchestration/blob/master/docs/INSTALL.md to install OpenSDS and all the required components of Orchestration.
+Please refer https://github.com/opensds/orchestration/blob/master/docs/INSTALL.md to install SODA and all the required components of Orchestration.
```
Comment (kumarashit) [inline content/guides/user-guides/orchestration/_index.md:25]:
```suggestion
Please refer https://github.com/sodafoundation/orchestration/blob/master/docs/INSTALL.md to install SODA and all the required components of Orchestration.
```

File: content/guides/user-guides/orchestration/_index.md [hunk 7]
```diff
@@ -87,8 +87,8 @@ CLI for Orchestration
 
 ### Steps to run CLI:
 
-* Deploy OpenSDS (https://github.com/opensds/opensds/wiki)
-* Deploy StackStorm with [OpenSDS packs](https://github.com/opensds/orchestration/blob/master/docs/INSTALL.md)
+* Deploy SODA (https://github.com/opensds/opensds/wiki)
```
Comment (kumarashit) [inline content/guides/user-guides/orchestration/_index.md:90]: can we provide the link to docs.sodafoundation. we are going to phase out wiki

File: content/guides/user-guides/orchestration/_index.md [hunk 8]
```diff
@@ -87,8 +87,8 @@ CLI for Orchestration
...
+* Deploy StackStorm with [SODA packs](https://github.com/opensds/orchestration/blob/master/docs/INSTALL.md)
```
Comment (kumarashit) [inline content/guides/user-guides/orchestration/_index.md:91]:
```suggestion
* Deploy StackStorm with [SODA packs](https://github.com/sodafoundation/orchestration/blob/master/docs/INSTALL.md)
```

**Review by kumarashit — CHANGES_REQUESTED** [review]

**Review by kumarashit — COMMENTED** [review]
LGTM

**Pull request merged**